release-notes/EPM/I-DOCSIS/EPM_6.1.10_I-DOCSIS.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: EPM_6.1.10_I-DOCSIS
+---
+
+# EPM 6.1.10 I-DOCSIS - preview
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be modified or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### Topology chains and chain fields can now be hidden [ID_37944]
+
+It is now possible to hide specific chains, chain fields, search chains, search chain tabs, and search chain fields in the I-DOCSIS EPM UI in DataMiner Cube. You can do so on the *Configuration* > *Chain and Field Settings* page of the EPM front-end element.
+
+## Changes
+
+### Enhancements
+
+#### Dashboard folders adjusted and unused dashboards removed [ID_37767]
+
+Several unused dashboards have been removed, along with the *EPM Solution* folder containing them. In addition, two new folders, *CCAP* and *Network*, have been added, each containing the respective overview pages. Finally, the Service Group Overview dashboard has been moved to the *SERVICE GROUP* folder.
```

Comment:
@EstebanSkyline, could you provide a list of the dashboards that have been removed, so I can add this to this RN entry?

Reply:
Sure ! 
The removed Dashboards are: 
I-DOCSIS CCAP -Trend_Test.dmadb
I-DOCSIS GLOBAL SYSTEM COUNTS.dmadb